apps/server/src/modules/files/repo/embedded-files.repo.ts

```diff
@@ -1,29 +1,27 @@
 /* istanbul ignore file */
+import { FilterQuery } from '@mikro-orm/core';
 import { EntityManager, ObjectId } from '@mikro-orm/mongodb';
 import { Injectable } from '@nestjs/common';
-import { File, FileRecordParentType, Lesson } from '@shared/domain';
+import { Counted, File, FileRecordParentType, Lesson, Task } from '@shared/domain';
 import { SyncFileItemMapper } from '../mapper';
-import { LessonMapper } from '../mapper/lesson-mapper';
 import { SyncFileItem } from '../types';
 
 export const fileUrlRegex = '"(https?://[^"]*)?/files/file\\?file=';
-const lessonsQuery = [
-	{
-		$match: {
-			'contents.component': {
-				$eq: 'text',
-			},
-		},
-	},
-	{
-		$match: {
-			'contents.content.text': {
-				$regex: new RegExp(`src=${fileUrlRegex}`),
-				$options: 'i',
-			},
-		},
-	},
-];
+const fileIdRegex = '(0x|0h)?[0-9A-F]{24}';
```

Comment:
(0x|0h) o_O ...i have no idea what it is do. :D
Maybe you can explain it me in meeting.